package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "smart-home-testing",
+  "version": "1.0.0",
+  "description": "Testing library for Smart Home Actions for developers to create and run integration tests locally. ",
+  "main": "src/validate.ts",
+  "scripts": {
+    "test": "mocha",
+    "compile": "tsc",
+    "check": "gts check",
+    "clean": "gts clean",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/googleinterns/smart-home-testing.git"
+  },
+  "author": "Jolene Lozano",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/googleinterns/smart-home-testing/issues"
+  },
+  "homepage": "https://github.com/googleinterns/smart-home-testing#readme",
+  "devDependencies": {
+    "chai-json-schema-ajv": "^5.1.0",
+    "mocha": "^8.0.1",
+    "mocha-param": "^2.0.1",
+    "ts-node": "^8.10.2",
```

Comment:
Is this used? Planning to use?

Reply:
This is here for the example developer test - could remove this though for this PR

Reply:
Yeah we should make each commit atomic, only including what is required. Later dependencies can be included in the future alongside an example developer test.